worker/worker/handler.py

```diff
@@ -61,13 +63,15 @@ def process(sess: Session, message: SQSMessageRecord, nonce: int = None) -> Tupl
         .where(Product.id == message.body.get("product_id"))
     )
 
-    fav_data = [{
-        "balanceAddr": message.body.get("agent_addr"),
-        "value": {
-            "currencyTicker": x.ticker.value,
-            "value": x.amount,
-        }
-    } for x in product.fav_list]
+    planet_id: PlanetID = PlanetID(bytes(message.body["planet_id"], 'utf-8'))
+    agent_address = message.body.get("agent_addr")
+    avatar_address = message.body.get("avatar_addr")
+    # relay
+    if planet_id != PlanetID.ODIN:
+        vault_address = derive_vault_address(planet_id).hex()
+        agent_address = vault_address
+        avatar_address = vault_address
```

Comment:
Is it OK to send agent-bound FAVs and avatar-bound inventory items to same address?
Because I don't know much about the `vault` and `relay`.

Reply:
@longfin That's what I understand, is that correct?